manifests/params.pp

```diff
@@ -40,7 +40,10 @@
 
     'RedHat': {
       $dhcp_dir    = '/etc/dhcp'
-      $packagename = 'dhcp'
+      $packagename = $facts['os']['release']['major'] ? {
+        '7'     => 'dhcp',
+        default => 'dhcp-server',
+      }
```

Comment:
We also support EL6 according to metadata.json so you'd break that. That's why we commonly use version comparisons:
```suggestion
      if versioncmp($facts['os']['release']['major'], '8') >= 0 {
        $packagename = 'dhcp-server'
      } else {
        $packagename = 'dhcp'
      }
```

Reply:
I removed EL6 from metadata.json. Is there a real reason to continue supporting it @ekohl ?

I will make the change to use versioncmp.

Reply:
Normally we consider that a breaking change. If you're going to drop EL6, you also need to remove all code for it, otherwise there's no point. You generally don't do that in the same PR if you can avoid it to keep things focused. That makes review easier and in turn gives it a greater chance of being merged faster. https://theforeman.org/2019/03/merge-time-review-complexity.html is some data on this.

Reply:
This comment is still relevant since I think effectively you're breaking EL6 support.

Reply:
Pushed changes to implement versioncmp and maintain functionality on EL6